cg/apps/cgstats/stats.py

```diff
@@ -145,6 +145,8 @@ def fastqs(self, flowcell: str, sample_obj: Sample) -> Iterator[Path]:
         """Fetch FASTQ files for a sample."""

```

Comment:
I think it would be helpful to identify what pattern is used and needed by what and update the docstring.
It will make our lives easier in future when we hopefully restructure this.

Reply:
I agree, might be out of scope for this PR though. Do you have any ideas for a good doc string?

Reply:
I think the doc string should not leak too much of the implementation details. But maybe something like
```
Retrieve all fastq files for a specific sample in a flow cell directory.
```

Reply:
Something along the lines of:

root_pattern is used for the flat output structure for the novaseqx sequencer types
unaligned pattern is used for the .... structure  for the xx sequencer types 
etc. 

I think as soon as we have implemented the new flow properly we can revisit the patterns and remove them. So I guess we either do it now or later. Up to you I suppose

Reply:
Alternatively, we could refactor this function later to take in a parameter indicating which bcl software was used to make the logic clearer in the function

Reply:
Yeah, I can try to add comments about that in the function

Reply:
Added some comments to clarify what the different patterns represent, check whether they make sense - I'm not familiar with the different patterns

Reply:
Updated the doc strings to be clearer

Reply:
I think these changes are out of scope of this PR but might as well do it now 👍